chapters/data-collection.tex

```diff
@@ -105,50 +107,65 @@ \subsection{Data licensing agreements}
 and to fully inform others of what you are doing.
 Writing down and agreeing to specific details is a good way of doing that.
 
-When you are licensing someone else's data for research,
-keep in mind that they are not likely to be familiar
+If the research team requires access to existing data for novel research,
+terms of use should be agreed on with the data owner, 
+typically through a data licensing agreement.
+Keep in mind that the data owner is likely not familiar
 with the research process, and therefore may be surprised
 at some of the things you want to do if you are not clear up front.
-You will typically want the right to create and retain
-derivative indicators, and you will want to own that output dataset.
-You will want to store, catalog, or publish, in whole or in part,
-either the original licensed material or the derived dataset.
+You will typically want intellectual property rights to all derivative works developed used the data,
```

Comment:
I am not sure what the extent of what we are saying here is. What is included in "derivative works"? Are those research outputs, or are they also data sets generated during cleaning/construct? 

Can we use a easier and more direct term here? What about?

```suggestion
You will typically want intellectual property rights to all research outputs developed used the data,
```